### PR TITLE
Docs: lcd160cr.rst get_line() describes usage.

### PR DESCRIPTION
**Pull request by peterhinch**
Improved description of the usage of this method.

### CODE REVIEW

**Comment by peterhinch** [comment]
I have added a comment re JPEG data format in the light of issue #2914.

File: docs/library/lcd160cr.rst [hunk 1]
```diff
@@ -341,7 +343,8 @@ Advanced commands
 
 .. method:: LCD160CR.jpeg(buf)
 
-    Display a JPEG.  `buf` should contain the entire JPEG data.
+    Display a JPEG.  `buf` should contain the entire JPEG data. JPEG data should
```
Comment (chuckbook) [inline docs/library/lcd160cr.rst]: Maybe we can add a note about the supported encoding.
`Baseline DCT, Huffman coding, 8 bits per sample, 3 color components, YCbCr4:2:2`

Comment (peterhinch) [inline docs/library/lcd160cr.rst]: Now done.

**Comment by dpgeorge** [comment]
Thanks, I squished and merged the 2 commits for the JPEG encoding, in 1f549a3496ac543390a170a7eb7b242475063016.

The get_line method may be cleaned up further and/or removed, so let's document that at a later date.

**Pull request closed without merging**